contracts/sui/contracts/sources/stork.move

```diff
@@ -0,0 +1,172 @@
+module stork::stork {
+
+    // === Imports ===
+
+    use stork::admin::AdminCap;
+    use stork::state::{Self, StorkState};
+    use stork::temporal_numeric_value_feed::{Self, TemporalNumericValueFeed};
+    use stork::temporal_numeric_value::TemporalNumericValue;
+    use stork::encoded_asset_id::{Self, EncodedAssetId};
+    use stork::verify::verify_stork_evm_signature;
+    use stork::event::Self;
+    use stork::update_temporal_numeric_value_evm_input::UpdateTemporalNumericValueEvmInput;
+    use stork::update_temporal_numeric_value_evm_input_vec::UpdateTemporalNumericValueEvmInputVec;
+    use sui::sui::SUI;
+    use sui::coin::Coin;
+    use sui::object_table::ObjectTable;
+
+    // === Errors ===
+
+    const EInvalidSignature: u64 = 0;
+    const EInsufficientFee: u64 = 1;
+    const EFeedNotFound: u64 = 2;
+
+    // === Entry Functions ===
+
+    entry fun init_stork(
+        // admin capability
+        _: &AdminCap,
+        // the address of the Stork program
+        stork_sui_public_key: address,
```

Comment:
Is this meant to represent the current contract address? If so, I'd suggest a more descriptive name.

Reply:
Why is this necessary to store on the contract?